### ISSUE TITLE
Include ECGAN into StudioGAN

### ISSUE THREAD

**Issue by sian-chen** [POSTECH-CVLab/PyTorch-StudioGAN#107]
Dear Authors,

Thank you for your great work.
We are happy to share with you that our paper [A Unified View of cGANs with and without Classifiers](https://arxiv.org/abs/2111.01035) is accepted by NeurIPS 2021.
We develop a new cGAN architecture called ECGAN, which achieves state-of-the-art results compared with current cGANs.
Our [implementation](https://github.com/sian-chen/PyTorch-ECGAN/) is based on StudioGAN, so it can be easily merged into your repository.
We appreciate your great work that allows us to develop new architecture much easier.
It would be my honor if ECGAN can be included in StudioGAN.
If you are interested, I can prepare a pull request.
What do you think?

Best Regards,
Si-An

**Comment by MINGUKKANG** [comment]
Hi Si-An,

Congratulation for your great work. 

Yesterday, I see your paper were published  at arXiv, and It is great pleasure that StudioGAN library helps you to study various cGAN more effectively.

**Comment by MINGUKKANG** [comment]
Hi Si-An,

Congratulations on your great work being published in Neurips 2021. 

I saw your paper (ECGAN) published in arXiv yesterday, and I am so happy since I remember you emailed me to ask questions regarding ContraGAN (publishing paper in arXiv or Neurips means you achieve significant improvement).

It is also a great honor for us if you make a pull request to include your wonderful research output into the StudioGAN library.

Unfortunately, StudioGAN has undergone a big version change, and almost all codes are different from the previous implementations. Thus, if you are willing to add your implementation into StudioGAN, we are welcome your contribution.

Thank you.

Best,

Minguk

**Comment by sian-chen** [comment]
So glad to know that you remember me.
I see the differences. It may take some time but I'm still willing to contribute to the library.
I'll let you know once the PR is ready. Thanks!

**Comment by MINGUKKANG** [comment]
Thank you so much. 

I hope to see you at an offline conference someday.
